### PR TITLE
Clothing changes, ECL/ITDO addons, new stamps, command pets

### PR DESCRIPTION
**Pull request by ChromeFoxxity**
## About the Pull Request

Adds a ECL uniform, and also puts the stuff in the locker manually without just being mapped in. Also adds Police Jacket for 912. As well as also adds heads of staff/representative pets, new stamp pathtypes, titanium poly windows, changes suits pathtypes, removed Office Worker's medical access, and more science access, removed some EZ Medical access because it's mostly unused, and or used by AMD. 

## Why It's Good For The Game

To look better!

## Changelog

:cl:
add: ECL Uniform, ECL Jacket, ITDO Uniform, Police Jacket. Also adds lockers. Head of Staff/Rep Pets. New Stamps. Titanium Poly Windows. Changes Suit pathypes. Adds suitskirts.
del: Office Worker's medical access. Lowers EZ Medical Access because it's unused. Removes unnecessary suits.
/:cl:

### CODE REVIEW

File: code/game/jobs/job/misc.dm [hunk 1]
```diff
@@ -141,9 +141,6 @@
 		ACCESS_CIV_COMMS,
 		ACCESS_ADMIN_LVL1,
 		ACCESS_SCIENCE_LVL1,
-		ACCESS_SCIENCE_LVL2,
```
Comment (projectkepler-ru) [inline code/game/jobs/job/misc.dm:144]: DOCUMENT THESE CHANGES

File: code/game/jobs/job/security.dm [hunk 2]
```diff
@@ -195,7 +195,6 @@
 		ACCESS_MEDICAL_LVL1,
 		ACCESS_MEDICAL_LVL2,
 		ACCESS_MEDICAL_LVL3,
-		ACCESS_MEDICAL_LVL4,
```
Comment (projectkepler-ru) [inline code/game/jobs/job/security.dm:204]: DO NOT FUCKING REMOVE THIS ACCESS

File: code/game/jobs/job/security.dm [hunk 3]
```diff
@@ -387,8 +386,6 @@
 		ACCESS_SEC_COMMS,
 		ACCESS_SCIENCE_LVL1,
 		ACCESS_MEDICAL_LVL1,
-		ACCESS_MEDICAL_LVL2,
```
Comment (projectkepler-ru) [inline code/game/jobs/job/security.dm:396]: DO NOT FUCKING REMOVE THIS ACCESS

File: code/game/jobs/job/security.dm [hunk 4]
```diff
@@ -537,8 +534,6 @@
 		ACCESS_SECURITY_LVL2,
 		ACCESS_ENGINEERING_LVL1,
 		ACCESS_MEDICAL_LVL1,
-		ACCESS_MEDICAL_LVL2,
```
Comment (projectkepler-ru) [inline code/game/jobs/job/security.dm:546]: DO NOT FUCKING REMOVE THIS ACCESS

File: code/modules/clothing/suits/jobs.dm [hunk 5]
```diff
@@ -41,13 +41,34 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	armor = list(melee = 90, bullet = 90, laser = 90, energy = 90, bomb = 80, bio = 40, rad = 60)
 
-/obj/item/clothing/suit/armor/itd
+/obj/item/clothing/suit/armor/itdo
```
Comment (projectkepler-ru) [inline code/modules/clothing/suits/jobs.dm:44]: why the o

File: code/modules/clothing/under/miscellaneous.dm [hunk 6]
```diff
@@ -809,9 +809,37 @@
 	worn_state = "kimono"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 
-/obj/item/clothing/under/itd
+/obj/item/clothing/under/itdofficial
```
Comment (projectkepler-ru) [inline code/modules/clothing/under/miscellaneous.dm:822]: why the fuck is the naming inconsistent

File: code/modules/clothing/under/miscellaneous.dm [hunk 7]
```diff
@@ -809,9 +809,37 @@
...
 	desc = "It's a snazzy dark purple formal shirt worn by someone who works in the \"Internal Tribunal Department\", it has silver markings and a white nametag."
-	name = "Internal Tribunal uniform"
+	name = "Internal Tribunal Official uniform"
```
Comment (projectkepler-ru) [inline code/modules/clothing/under/miscellaneous.dm:824]: we don't use the  word official in this organization in reference to a person who act as representative of their branch

**Comment by checkraisefold** [comment]
image
???? what the ants doing

File: code/modules/clothing/under/miscellaneous.dm (hunk 7 above)
Comment (ChromeFoxxity) [reply code/modules/clothing/under/miscellaneous.dm:824]: > we don't use the word official in this organization in reference to a person who act as representative of their branch

Meant to be for those who report to the site, like full on ITD Officials.

File: code/modules/clothing/under/miscellaneous.dm (hunk 6 above)
Comment (ChromeFoxxity) [reply code/modules/clothing/under/miscellaneous.dm:822]: It's just a uniform.

File: code/modules/clothing/suits/jobs.dm (hunk 5 above)
Comment (ChromeFoxxity) [reply code/modules/clothing/suits/jobs.dm:44]: For ITD "Officer"?

**Comment by Ambrose997** [comment]
A bit sly don't you think? You didn't mention the nerfing of ITDO access.

Says in the SOP that his power is equal to that of the EZ Zone Commander, doesn't make sense to lower his access whatsoever.

File: code/game/jobs/job/security.dm [hunk 8]
```diff
@@ -388,7 +387,6 @@
 		ACCESS_SCIENCE_LVL1,
 		ACCESS_MEDICAL_LVL1,
 		ACCESS_MEDICAL_LVL2,
-		ACCESS_MEDICAL_LVL3,
```
Comment (checkraisefold) [inline code/game/jobs/job/security.dm:397]: the only problem I have with this is that the crematorium requires medical lvl3 and ez sec might need to use it. maybe keep it for senior agent *or* change the crematorium button to level 2

File: tools/UpdatePaths/Scripts/65236_clothingunderrepath.txt
Comment (checkraisefold) [inline tools/UpdatePaths/Scripts/65236_clothingunderrepath.txt:1]: remove the changes from this file

File: maps/site53/structures/closets/research.dm [hunk 9]
```diff
@@ -18,7 +18,7 @@
 		/obj/item/clothing/under/rank/research_director,
 		/obj/item/clothing/under/rank/research_director/rdalt,
 		/obj/item/clothing/under/rank/research_director/dress_rd,
-//		/obj/item/clothing/under/suit_jacket/nt,
+//		/obj/item/clothing/under/suit/nt,
```
Comment (checkraisefold) [inline maps/site53/structures/closets/research.dm:21]: remove this change

File: maps/site53/site53areas.dm [hunk 10]
```diff
@@ -616,13 +616,13 @@
 	turf_initializer = /decl/turf_initializer/maintenance
 
 /area/site53/uez/repoffice/internaltribunal
-	name = "\improper Internal Tribunal Departments Office"
+	name = "\improper Internal Tribunal Department Officer's Office"
```
Comment (checkraisefold) [inline maps/site53/site53areas.dm:631]:
```suggestion
	name = "\improper Internal Tribunal Officer's Office"
```
acceptable short name

File: code/modules/mob/living/simple_animal/friendly/ant.dm [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!ckey && !stat)
+		if(isturf(src.loc) && !resting && !buckled)		//This is so it only moves if it's not inside a closet, gentics machine, etc.
+			turns_since_move++
+			if(turns_since_move >= turns_per_move)
+				Move(get_step(src,pick(4,8)))
+				turns_since_move = 0
+	regenerate_icons()
```
Comment (checkraisefold) [inline code/modules/mob/living/simple_animal/friendly/ant.dm:44]: huh?? this is what the AI holder is for. remove this shitcode

File: code/game/objects/structures/window.dm [hunk 12]
```diff
@@ -675,6 +675,12 @@
 	dir = 5
 	icon_state = "rwindow_full"
 
+/obj/structure/window/reinforced/polarized/titanium
+	name = "electrochromic window"
+	desc = "Adjusts its tint with voltage. Might take a few good hits to shatter it."
+	basestate = "rwindow"
+	polarized = 1
+
```
Comment (checkraisefold) [inline code/game/objects/structures/window.dm:683]: umm you didnt change anything or the material its the same as the default polarized window

File: code/game/objects/structures/wallframe_spawner.dm [hunk 13]
```diff
@@ -170,6 +170,11 @@
 	name = "polarized wall frame window spawner (no grille) (non reinforced)"
 	win_path = /obj/structure/window/basic/full/polarized
 
+/obj/effect/wallframe_spawn/reinforced/titanium/polarized
+	name = "polarized reinforced titanium wall frame window spawner"
+	color = "#444444"
+	win_path = /obj/structure/window/reinforced/polarized/full
+
```
Comment (checkraisefold) [inline code/game/objects/structures/wallframe_spawner.dm:177]: doesnt mention that its a window in the name

**Review by checkraisefold — CHANGES_REQUESTED** [review]

File: tools/UpdatePaths/Scripts/65236_clothingunderrepath.txt
Comment (checkraisefold) [inline tools/UpdatePaths/Scripts/65236_clothingunderrepath.txt:1]: remove the changes from this file (2)

**Review by checkraisefold — CHANGES_REQUESTED** [review]

**Comment by checkraisefold** [comment]
Closing this until interest in updating it is expressed

**Pull request closed without merging**